frontend/src/pages/Alerts/utils/AlertsUtils.ts

```diff
@@ -58,22 +58,26 @@ export interface AlertForm {
  */
 export const findAlert = (
 	id: string,
+	type: 'error' | 'session',
```

Comment:
Added to avoid id conflict - Alert and session alerts can have the same `id` so this could end up returning the wrong alert